CliClient/tests/md_to_md/sample.md

```diff
@@ -0,0 +1,10 @@
+# Markdown
+
+lorem ipsum ![alt text here](joplin_logos.png)
+- [ ] check!
+- [ ] boxes!
+
+![alt text here](joplin_logos.png)ipsum lorem
+
+**strong text**
+![does not exist](does_not_exist.png) lorem ipsum
```

Comment:
For test external assets, we generally place them in tests/support. In your case, you can simply reuse `tests/support/photo.jpg` (and that means deleting md_to_md/joplin_logos.png)

Reply:
done